content/en/real_user_monitoring/guide/alerting-with-rum.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Alerting with RUM Data
+kind: guide
+description: Guide for creating alerts on RUM events.
+further_reading:
+    - link: '/real_user_monitoring/dashboards/'
+      tag: 'Documentation'
+      text: 'RUM Dashboards'
+    - link: '/monitors/create/types/real_user_monitoring/'
+      tag: 'Documentation'
+      text: 'RUM Monitor'
+---
+
+## Overview
+
+Real User Monitoring allows you to create alerts which notify you about anomalous behavior in your applications. You can create RUM monitors with complex conditions, predefined thresholds, and multiple queries to calculate averages, ratios, and performance indicator metrics (such as Apdex).
```

Comment:
Instead of anomalous behavior, let's go with atypical behavior

Reply:
```suggestion
Real User Monitoring allows you to create alerts which notify you about atypical behavior in your applications. You can create RUM monitors with complex conditions, predefined thresholds, and multiple queries to calculate averages, ratios, and performance indicator metrics (such as Apdex).
```